src/main/java/com/cinema/annotations/FieldsValueMatchValidator.java

```diff
@@ -0,0 +1,24 @@
+package com.cinema.annotations;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import org.springframework.beans.BeanWrapperImpl;
+
+public class FieldsValueMatchValidator
+        implements ConstraintValidator<FieldsValueMatchValidation, Object> {
+    private String field;
+    private String fieldMatch;
+
+    @Override
+    public void initialize(FieldsValueMatchValidation constraintAnnotation) {
+        this.field = constraintAnnotation.field();
+        this.fieldMatch = constraintAnnotation.fieldMatch();
+    }
+
+    @Override
+    public boolean isValid(Object dto, ConstraintValidatorContext constraintValidatorContext) {
```

Comment:
dto is not correct naming if you want to make this check general 
so either parameterize it with UserRequestDto or change variable name

Reply:
used "object" as variable name